addons/web/static/src/core/tooltip/tooltip.scss

```diff
@@ -46,4 +46,9 @@
     + .popover-arrow {
         --popover-arrow-color: #{$tooltip-arrow-color};
     }
+
+    + .popover-arrow::after {
+        border-bottom-color: #{$tooltip-arrow-color};
+        border-top-color: #{$tooltip-arrow-color};
+    }
```

Comment:
We are already using this tooltip where those lines are not needed. It should work without them 🤔
Investigate why it works with this tooltip and not with the new ones...
![image](https://github.com/user-attachments/assets/7e3d8464-7f5b-4d2e-8d99-919f79999d0b)